docs/api_v0.yml

```diff
@@ -2790,6 +2831,54 @@ paths:
               -d '{"listing":{"title":"Title"}' \
               https://dev.to/api/listings/{id}
 
+  /readinglist:
+    get:
+      operationId: getReadinglist
+      summary: User's readinglist
+      description: |
+        This endpoint allows the client to retrieve a list of the reactions with the type readinglist along with the related article for the authenticated user.
+
+        "Articles" are all the posts that users create on DEV that typically
+        show up in the feed. They can be a blog post, a discussion question,
+        a help thread etc. but is referred to as article within the code.
```

Comment:
I think we can do away with this as we're sendinb back reading list items, not articles